gpjax/gps.py

```diff
@@ -198,16 +195,16 @@ class ConjugatePosterior(AbstractPosterior):
     jitter: Optional[float] = DEFAULT_JITTER
 
     def predict(
-        self, train_data: Dataset, params: Dict
-    ) -> Callable[[Float[Array, "N D"]], dx.Distribution]:
+        self, train_data: Dataset, params: dict
+    ) -> Callable[[Float[Array, "N D"]], npd.Distribution]:
```

Comment:
Dictionary typing needs to be consistent - your changes have replaced some methods with `Dict` to `dict`, while others are unchanged . I opt to us `Dict` as it more general, though happy to switch to `dict` if there's good reason - though all methods need to be changed throughout the library.

Reply:
These have been reolved now.